spec/string_foundation/length_spec.rb

```diff
@@ -0,0 +1,64 @@
+# ==============================================================================
+# SPEC - STRING FOUNDATION - LENGTH
+# ==============================================================================
+# frozen_string_literal: true
+describe "[ Length Methods ]" do
+
+  # ----------------------------------------------------------------------------
+  # Compare Characters Length
+  # ----------------------------------------------------------------------------
+  describe "COMPARE CHARACTERS LENGTH" do
+    let(:string) { "string" }
+    let(:expected_length) { 0 }
+    subject { string.length?(expected_length) }
+
+    context "when a argument is String class," do
+      let(:expected_length) { "6" }
+
+      it { is_expected_as_block.to raise_error ArgumentError }
```

Comment:
Wrap put parentheses `()` around for arguments like `raise_error(ArgumentError)` .